C/Bresenham-Circle.c

```diff
@@ -0,0 +1,58 @@
+#include <stdio.h>
```

Comment:
Please add some documentation in the form of comments to explain what the Bresenham Algorithm is all about, why it is faster than the midpoint algorithm